doc/STYLE_GUIDE_RSPEC.md

```diff
@@ -153,3 +153,16 @@ end
 ```
 
 **Note:** this is a Part of Rspec configuration. Read more [by link](https://github.com/rubocop-hq/rubocop-rspec#enforcing-an-explicit-rspec-receiver-for-top-level-methods-disabling-monkey-patching).
+
+* <a name="rspec-let-setup"></a>
+  We allow using `let!` in your test
```

Comment:
We allow using `let!` in your test, but suggest not to use it for setting up testing data